theano/sandbox/jaxify.py

```diff
@@ -984,3 +985,16 @@ def ravelmultiindex(*inp, mode=mode, order=order):
         return jnp.ravel_multi_index(multi_index, dims, mode=mode, order=order)
 
     return ravelmultiindex
+
+
+# TODO: Remove all these when done
+# https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.eye.html#jax.numpy.eye
+@jax_funcify.register(Eye)
+def jax_funcify_Eye(op):
+    # order = op.order
```

Comment:
`Eye` has a [`dtype`](https://github.com/pymc-devs/Theano-PyMC/blob/master/theano/tensor/basic.py#L2946) property that you'll need to set here.

Reply:
Do I set dtype like this? https://github.com/pymc-devs/Theano-PyMC/blob/master/theano/sandbox/jaxify.py#L237

Reply:
Yes, or you can use the same idiom as the commented line:
```python
def jax_funcify_Eye(op):
    dtype = op.dtype

    def eye(N, M, k, dtype=dtype):
        ....
```